specification.rst

```diff
@@ -47,6 +47,13 @@ throughout this specification when referring to CSL elements, but is generally
 omitted in favor of a default namespace declaration (set with
 the ``xmlns`` attribute) on the root ``cs:style`` or ``cs:locale`` element.
 
+Whitespace Handling
+-------------------
+
+CSL styles are valid XML, but CSL processors MUST NOT normalize attribute values 
+by trimming leading or trailing whitespace from attributes which define text that 
+is intended for output: delimiter="," and delimiter=" ," are distinct values."
```

Comment:
Lgtm. Just one question: It should be clear, but perhaps we should be more explicit that this not only applies to `@delimiter`?
What about: "e.g., delimiter="," and delimiter=" ," are distinct values."

Reply:
I don't think it's just delimiters. I think it is also `prefix` `suffix` `range-delimiter` and `initialize-with` ... at least those!

Reply:
OK, I added the full list of attributes identified by @PaulStanley and @bwiernik 

RTM?